virtwho/register.py

```diff
@@ -341,14 +341,16 @@ def pool_id_get(self, sku_id, sku_type='Physical'):
 
 class RHSM:
 
-    def __init__(self):
+    def __init__(self, rhsm='rhsm'):
```

Comment:
there will be more rhsm session in the virtwho.ini, such as `[rhsm]` and `[rhsm_sw]`, and we may add more accounts, so add the param to make it clever.